src/modules/getGameScores.js

```diff
@@ -0,0 +1,23 @@
+import getGameID from './getGameID.js';
+import showUserScores from './showUserScores.js';
+
+const id = getGameID();
+const url = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores/`;
+
+const getGameScores = async () => {
+  try {
+    const response = await fetch(url);
+
+    if (!response.ok) {
+      throw new Error('Network response was not ok');
+    }
+
+    const data = await response.json();
+    const userScores = data.result;
+    showUserScores(userScores);
+  } catch (error) {
+    // return error;
+  }
```

Comment:
- [ ] Please consider removing the commented code from our codebase to maintain a clean and professional code repository.

- [OPTIONAL] I wanted to suggest a slight improvement in our code. Instead of leaving the `catch` block empty, consider throwing an error within it. This way, you can proactively handle the error at a later point in the code.